lgc/patch/PatchEntryPointMutate.cpp

```diff
@@ -1190,6 +1190,62 @@ void PatchEntryPointMutate::addUserDataArgs(SmallVectorImpl<UserDataArg> &userDa
     return;
   }
 
+  // Define the function
```

Comment:
This comment is not very helpful. It'd be better to comment what this lambda is used for, why do we need it, etc.